README.md

```diff
@@ -94,6 +94,36 @@ Set `absoluteFilePathsForReporter` to `true` to use absolute file paths in gener
 [getting_started]: http://gruntjs.com/getting-started
 [vnujar]: https://validator.github.io/validator/
 
+### `partialWrapperTemplatePath`
+
+Type: `String`
+Default: `null`
+
+Provide a path to wrap html partials for validation.
+
+```js
+all: {
+  options: {
+    wrapPath: 'config/htmlValidationWrapper.html'
```

Comment:
Could also just name this `layout` or something similar. This is basically just implementing an already well established pattern from the view portion of many frameworks.